.github/workflows/matlab_tests_and_docs.yml

```diff
@@ -20,9 +20,9 @@ jobs:
           command: cd docs, cd src, bringONSASmFilesToONSASdocs
       - uses: julia-actions/setup-julia@latest
         with:
-          version: 1.8
+          version: 1
```

Comment:
This will fetch the latest Julia image, for context you can take a look at [this comment](https://github.com/JuliaLang/Example.jl/pull/68#discussion_r995947019)

Reply:
👍